src/storage/mod.rs

```diff
@@ -386,6 +416,10 @@ pub struct Options {
     pub skip_constraint_check: bool,
     pub key_only: bool,
     pub reverse_scan: bool,
+    pub is_first_lock: bool,
+    pub should_not_exist: bool,
+    pub is_pessimistic_lock: Vec<bool>,
+    pub prewrite_pessimistic_lock: bool,
```

Comment:
Can we remove this option since we separated functions?

Reply:
yes